Tasks/WindowsMachineFileCopyV2/RoboCopyJob.ps1

```diff
@@ -101,7 +106,12 @@ param (
 
     function Clean-Target
     {
-        $cleanupArgument = "/NOCOPY /E /PURGE" 
+        if(-not $ModifyRoboCopyRetries){
+          $cleanupArgument = "/NOCOPY /E /PURGE" 
+        }
+        else{
+          $cleanupArgument = "/NOCOPY /E /PURGE /R:3 " 
```

Comment:
Same here: it will be great to have an opportunity to pass number of retries from the backend (for ex, $env:CUSTOM_NUMBER_OF_RETRIES_IN_ROBOCOPY)

Reply:
There is an additional argument input in the task where customers can specify their desired retry values. If they enable the feature flag, the task will use 3 retries by default; otherwise, it will default to 1 million retries.
There is another case where a customer enables the feature flag (resulting in 3 retries by default) and also specifies 5 retries in the additional argument. In this case, the task will use 5 retries as the provided value takes precedence.
![image](https://github.com/user-attachments/assets/01621ea7-b238-4ab0-bb22-8b4d41b58cae)